lib/views/help/privacy.html.erb

```diff
@@ -2,573 +2,895 @@
+    for sensitive information or large volumes of breached personal data we
+    would appreciate being notified by anyone.
+  </p>
+  <p>
+    The General Data Protection Regulation gives you the right to withdraw
+    consent for the processing of your personal information; however, it also
+    gives us the right to refuse to comply with such requests on grounds
+    including, for example, that the data is being processed for ‘archiving
+    purposes in the public interest, scientific research historical research or
+    statistical purposes’.
+  </p>
+  <h4>
+    Removal of public servants’ personal information from responses
+  </h4>
+  <p>
+    WhatDoTheyKnow strives to preserve and promote transparency and openness. We
+    automatically publish responses to requests made through the site in full.
+    Where these responses include the personal information of public servants,
+    we consider the legal basis for our processing of this information to be
+    that publication is in the public interest (as described in
+    <a href="https://gdpr-info.eu/art-6-gdpr/">Article 6(1)(e)</a>
+    of the GDPR). We will consider requests to remove the names of public
+    servants when it seems that the interests, rights and freedoms of the data
+    subject outweigh the public interest in publishing.
+  </p>
+  <p>
+    This means that:
+  </p>
+  <ul>
+    <li>
+      If you are a decision-maker of any seniority, or if you are responding to an
+      FOI request, we will not normally remove your details from documents and
+      emails sent by a public body. Accountability of decision-making is at the
+      heart of good government.
+    </li>
+    <li>
+      If you hold a junior, non-decision making post we will consider requests
+      to remove your details. Removing these details is time-consuming for our
+      volunteers, so please let us know why this really matters to you. If we
+      are not required to act by law but agree to remove your details we will
+      take reasonable steps to do so — but in some cases may not be able to as
+      it can be difficult to remove material from some types of document, such
+      as images and PDFs.
+    </li>
+  </ul>
+  <p>
+    We are happy try to assist FOI officers who want us to help them remove
+    their signatures from the website, but generally in such circumstances we
+    require replacement responses to be prepared and sent to the requester
+    before we will remove the original documents. We are generally unable to
+    edit attachments which have been released via our service.
+  </p>
+  <p>
+
+  <h3 id="right_to_access">
+    Your right to access
+    <a href="#right_to_access">#</a>
+  </h3>
+  <p>
+    You may contact us at any time to ask to see what personal data we hold
+    about you.  If you have used our service to make a request,  then the vast
+    majority of information we hold about you is shown publicly on our website.
```

Comment:
Could say something like "or in the user-only portions of pages" for e.g. things you're following / email address etc. Not a huge concern though.